dffml/record.py

```diff
@@ -3,6 +3,8 @@
 """
 Information on the software to evaluate is stored in a Record instance.
 """
+import os
+import numpy as np
```

Comment:
Yes you were correct, we want to keep the top level package free of dependencies

```suggestion
```